packages/wonder-blocks-tooltip/components/tooltip.md

```diff
@@ -1,30 +1,30 @@
-### Text anchor & text tooltip
+### Text anchor & text tooltip & placement right
 
 ```js
 const React = require("react");
 
-<Tooltip content={"The tooltip text"}>
+<Tooltip content={"I'm on the right!"} placement="right">
     Some text
 </Tooltip>
 ```
 
-### Complex anchor & text tooltip
+### Complex anchor & text tooltip & placement default (top)
 
 In this example, we're no longer forcing the anchor root to be focusable, since the text input can take focus.
 
 ```js
 const React = require("react");
 const {View} = require("@khanacademy/wonder-blocks-core");
 
-<Tooltip forceAnchorFocusivity={false} content={"The tooltip text"}>
+<Tooltip forceAnchorFocusivity={false} content={"I'm at the top!"}>
     <View>
```

Comment:
Maybe give this `View` a background color so that it's easier to see.

Reply:
It will have one when #159 is done.